website/docs/r/datadog_monitor_sso_configuration.html.markdown

```diff
@@ -50,7 +50,7 @@ The following arguments are supported:
 
 * `datadog_monitor_id` - (Required) The Datadog Monitor Id which should be used for this Datadog Monitor SSO Configuration. Changing this forces a new Datadog Monitor SSO Configuration to be created.
 
-* `single_sign_on_enabled` - (Required) The state of SingleSignOn configuration. Possible values are `Enable` and `Disable`.
+* `single_sign_on` - (Required) The state of SingleSignOn configuration. Possible values are `Enable`, `Disable`.
```

Comment:
You'll also want to add a note in the 5.0 upgrade guide

Reply:
Roger, added.